openflow_input/standard-1.4

```diff
@@ -1933,106 +1933,106 @@ struct of_table_feature_prop {
     uint16_t         length;
 };
 
-struct of_table_feature_prop_instructions(align=8, length_includes_align=False) : of_table_feature_prop {
```

Comment:
@abakagamze - I think you may have missed this one when you reverted the align changes?